backend/Epicenter.Persistence/Repositories/TimestampRepository.cs

```diff
@@ -10,26 +10,29 @@ namespace Epicenter.Persistence.Repositories
 {
     public class TimestampRepository : ITimestampRepository
     {
-        private readonly Mapper<Timestamp> _mapper;
+        private readonly EpicenterContext _context;
 
-        public TimestampRepository(Mapper<Timestamp> mapper)
+        public TimestampRepository(EpicenterContext context)
         {
-            _mapper = mapper;
+            _context = context;
         }
 
         public void Add(Timestamp entity)
         {
-            _mapper.CreateRow(entity);
+            _context.Timestamps.Add(entity);
+            _context.SaveChanges();
```

Comment:
good for now, later on it'll be dangerous to save context there (location logic), we'll will have to introduce unit of work